ctapipe-extra

```diff
@@ -1 +1 @@
-Subproject commit f7fe2209646dc8f99a28ec9ab4df8257687e1896
+Subproject commit 536490d27e34c5f3b0c361f828de208f7e17b60f
```

Comment:
yeah, that's a common problem. It will be better when ctapipe-extra is it's own package (we just need to migrate the code in ctapipe that uses it to call the correct function)